pebblo/app/service/local_ui_service.py

```diff
@@ -320,15 +330,25 @@ def get_all_apps_details(self):
                     )
             final_prompt_details = []
             for key, value in prompt_details.items():
-                for k1, v1 in value.items():
-                    prompt_dict = {}
-                    prompt_dict = {"app_name": key}
-                    prompt_dict["entity_name"] = k1
-                    prompt_dict["total_prompts"] = v1["total_prompts"]
-                    prompt_dict["total_entity_count"] = v1["total_entity_count"]
-                    prompt_dict["users"] = v1["users"]
-                    prompt_dict["total_users"] = v1["total_users"]
-                    final_prompt_details.append(prompt_dict)
+                try:
+                    for k1, v1 in value.items():
+                        try:
+                            prompt_dict = {}
+                            prompt_dict = {"app_name": key}
+                            prompt_dict["entity_name"] = k1
+                            prompt_dict["total_prompts"] = v1["total_prompts"]
+                            prompt_dict["total_entity_count"] = v1["total_entity_count"]
+                            prompt_dict["users"] = v1["users"]
+                            prompt_dict["total_users"] = v1["total_users"]
+                            final_prompt_details.append(prompt_dict)
+                        except Exception as ex:
+                            logger.error(
```

Comment:
Better to use warning instead of error in the exception within for loop.

Reply:
Done